tests/config/pytest_modules.yml

```diff
@@ -2644,6 +2644,40 @@ unicycler:
   - modules/nf-core/unicycler/**
   - tests/modules/nf-core/unicycler/**
 
+universc:
+  - modules/nf-core/universc/mkref/**
+  - modules/nf-core/universc/mkgtf/**
+  - modules/nf-core/universc/**
+  - tests/modules/nf-core/universc/mkref/**
+  - tests/modules/nf-core/universc/mkgtf/**
+  - tests/modules/nf-core/universc/**
+
```

Comment:
@Emiller88  That's a good point. I too noticed this when resolving issues with universc/mkfastq which was also run by testing universc. I created similar tests to the cellranger module (which you contributed):
https://github.com/nf-core/modules/blob/ca0cf339d05ccd0b69617bd8c76228570d01b8d3/tests/config/pytest_modules.yml#L484-L508
https://github.com/nf-core/modules/pull/979/files#diff-cdafd56bc0abcb70ff5cf745fa6afb6537f566748074cce623aca7eac3e8ba47

Since the necessary steps to generate the reference are included in tests/modules/nf-core/universc/main.nf I'll remove the redundant tests. As suggested above I'll also restructure the repo to avoid nested modules.